<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_cooperativa_alta</class>
 <widget class="QWidget" name="form_cooperativa_alta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>497</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Datos de Entidad</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;Courier 10 Pitch&quot;;
selection-color: rgb(0, 0, 0);
selection-background-color: rgb(170, 255, 127);
color: rgb(0, 0, 0);
background-color: rgb(117, 0, 88);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Datos de la Entidad</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Denominación:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_denominacion">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Matrícula Nacional N°:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_nro_matricula">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Reg. Provincial N°:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_nro_reg_prov">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Inicio de actividades:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDateEdit" name="dte_inicio_actividades">
           <property name="minimumSize">
            <size>
             <width>125</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>125</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="displayFormat">
            <string>dd/MM/yyyy</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>CUIT N°:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_coope_cuit">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>250</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Domicilio:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_coope_domicilio">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>N°:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_coope_nro_dom">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_23">
           <property name="text">
            <string>Piso:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_coope_piso">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="6" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="text">
            <string>C.Postal:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_coope_c_postal">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_28">
           <property name="text">
            <string>Localidad:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_coope_localidad">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="7" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_29">
             <property name="text">
              <string>Provincia:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lne_coope_provincia">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_30">
             <property name="text">
              <string>Telefono :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lne_coope_telefono">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>25</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>25</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="8" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QLabel" name="label_32">
           <property name="text">
            <string>Correo electrónico:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lne_coope_email">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
      <zorder></zorder>
      <zorder></zorder>
      <zorder></zorder>
      <zorder></zorder>
      <zorder></zorder>
      <zorder>line</zorder>
      <zorder></zorder>
      <zorder></zorder>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(226, 226, 226);</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Configuración Contable de la Cooperativa</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Capital  minimo obligatorio :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lne_cap_obliga">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(238, 238, 236);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_15">
             <property name="text">
              <string>Cantidad de cuotas:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lne_cant_cuotas">
             <property name="styleSheet">
              <string notr="true">background-color: rgb(238, 238, 236);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_13">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="boton_limpiar">
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(140, 189, 58);</string>
       </property>
       <property name="text">
        <string>Limpiar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boton_coope_actualizar">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(140, 189, 58);</string>
       </property>
       <property name="text">
        <string>Actualizar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="boton_coope_guardar">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>25</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(140, 189, 58);</string>
       </property>
       <property name="text">
        <string>GUARDAR</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_cant_cuotas</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>272</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_cap_obliga</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>472</y>
    </hint>
    <hint type="destinationlabel">
     <x>308</x>
     <y>372</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_coope_email</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>471</y>
    </hint>
    <hint type="destinationlabel">
     <x>249</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_coope_provincia</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>232</x>
     <y>255</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_coope_telefono</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>479</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_coope_c_postal</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>194</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_coope_domicilio</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>475</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_coope_nro_dom</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>297</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_coope_localidad</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>476</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_coope_piso</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>477</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_coope_cuit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>287</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_nro_matricula</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_nro_reg_prov</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>469</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>boton_limpiar</sender>
   <signal>clicked()</signal>
   <receiver>lne_denominacion</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>473</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>63</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
